StRoot/StTriggerUtilities/Bemc/StBemcTriggerSimu.cxx

```diff
@@ -44,7 +44,7 @@ StBemcTriggerSimu::StBemcTriggerSimu()
   mTables   = NULL;
   mHList    = NULL;
   mConfig   = 0;
-
+  mTestMode = false;
```

Comment:
```suggestion
```

Reply:
It says here

Update to the trigger simulator to save the intermediate jet patch sum ADCs and make them available to access for analysis. In this case save the 30 jet patch ADCs in the StJetSkimEvent.
Remove unnecessary printouts in the StJetMaker2012 during the UE offaxis cone calculation

Reply:
You only need to look at the jet maker

Reply:
Oops, this is a wrong suggestion. Looking through too quickly, I assumed this was an extra unused variable declaration.